frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1069,10 +1093,32 @@ export const funnelLogic = kea<funnelLogicType>({
             actions.loadEventWithPropertyCorrelations(eventName)
         },
 
-        excludeProperty: () => {
-            actions.setPropertyNames(
-                values.propertyNames.filter((property) => !values.excludedPropertyNames.includes(property))
-            )
+        excludePropertyFromProject: ({ propertyName }) => {
+            const oldExcludedPropertyNames = values.excludedPropertyNames
+            const oldCurrentTeam = teamLogic.values.currentTeam
+
+            if (oldCurrentTeam === null || oldExcludedPropertyNames === null) {
```

Comment:
Q: why do we return when `oldExcludedPropertyNames` are null? Doesn't that simply mean they haven't been set (yet) ?

Reply:
If we have not retrieved the old properties, we don't have enough info to set new ones as there is a dependency there.

Perhaps should log this as a warning? Or is there a better way to handle this? I suspect it's a case that doesn't actually happen with how the app actually loads?

Reply:
Ahhh, it's null when the currentTeam isn't loaded! My bad, yeah this is fine I guess.